### DISCUSSION TITLE
[Feature Request]: JSONL support processing (Newline Delimited JSON)

### DISCUSSION THREAD

**Discussion by iamriajul** [josdejong/svelte-jsoneditor#358] [Ideas]
More about JSONL available here: https://jsonlines.org/

Thanks.

**Comment by josdejong** [comment]
Thanks, that would be useful indeed. I see newline delimited JSON for example in log files. I guess it is the same as http://ndjson.org.

Help implementing this or thinking this through would be welcome.

For reference: there is also a feature request to support JSON variants, see https://github.com/josdejong/svelte-jsoneditor/issues/157

**Comment by josdejong** [comment]
Moving this idea to the [Discussions > Ideas](https://github.com/josdejong/svelte-jsoneditor/discussions/categories/ideas) section since we are not actively working on it.
